docs/aks.md

```diff
@@ -0,0 +1,177 @@
+# Install Percona Server for MongoDB on Azure Kubernetes Service (AKS)
+
+This guide shows you how to deploy Percona Operator for MongoDB on Microsoft
+Azure Kubernetes Service (AKS). The document assumes some experience with the
+platform. For more information on the AKS, see the [Microsoft AKS official documentation](https://azure.microsoft.com/en-us/services/kubernetes-service/).
+
+## Prerequisites
+
+The following tools are used in this guide and therefore should be preinstalled:
+
+1. **Azure Command Line Interface (Azure CLI)** for interacting with the different
+    parts of AKS. You can install it following the [official installation instructions for your system](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
+
+2. **kubectl**  to manage and deploy applications on Kubernetes. Install
+    it [following the official installation instructions](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+Also, you need to sign in with Azure CLI using your credentials according to the
+[official guide](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli).
+
+## Create and configure the AKS cluster
+
+To create your cluster, you will need the following data:
+
+* name of your AKS cluster,
+* an [Azure resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/overview), in which resources of your cluster will be deployed and managed.
+* the amount of nodes you would like tho have.
+
+You can create your cluster via command line using `az aks create` command.
+step). The following command will create a 3-node cluster named `my-cluster-name` within some [already existing](https://docs.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-cli#create-a-resource-group) resource group named `my-resource-group`:
+
+```bash
+$ az aks create -g my-resource-group -n my-cluster-name --enable-managed-identity --node-count 3 --node-vm-size Standard_B4ms --node-osdisk-size 30 --network-plugin kubenet  --generate-ssh-keys --outbound-type loadbalancer
+```
```

Comment:
I believe as example we can use here the whole name of the key like --name for example. It makes the command clearer.